src/Services/Processor/Substrate/Codec/Decoder.php

```diff
@@ -193,11 +192,7 @@ public function getValue(array $data, array $keys, ?string $prefix = null, ?stri
 
     public function collectionStorageData(string $data): array
     {
-        try {
-            $decoded = $this->codec->process('CollectionStorageDataV1010', new ScaleBytes($data));
-        } catch (\Exception $e) {
-            $decoded = $this->codec->process('CollectionStorageData', new ScaleBytes($data));
-        }
+        $decoded = $this->codec->process('CollectionStorageDataV1010', new ScaleBytes($data));
```

Comment:
Is this failing tests related to this fix?

Reply:
Yep, the tests are using the old format, I will fix the tests before we finish the PR